### PR TITLE
Document changes() and resets() functions.

### PR DESCRIPTION
**Pull request by juliusv**

### CODE REVIEW

**Comment by juliusv** [comment]
@fabxc @brian-brazil

File: content/docs/querying/functions.md [hunk 1]
```diff
@@ -40,6 +40,12 @@ by sample value.
 `ceil(v instant-vector)` rounds the sample values of all elements in `v` up to
 the nearest integer.
 
+## `changes()`
+
+For each input time series, `changes(v range-vector)` returns the number of
+times its value has changed within the provided time range as an instant
+vector.
```
Comment (fabxc) [inline content/docs/querying/functions.md]: Should we mention that it's fitted towards integers? For floats with small changes the result is fairly unpredictable.

**Comment by fabxc** [comment]
:+1: otherwise

File: content/docs/querying/functions.md (hunk 1 above)
Comment (juliusv) [inline content/docs/querying/functions.md]: Hmm, not sure. If two samples are ingested with the same value on the same machine, they will always end up having the same bit representation and be equal, no? The time where you have to be careful is when you do math on values before processing the result further, and slight orders in operations etc. could change the output value ever so slightly each time.

I can imagine this function being used with e.g. timestamps that are stored as seconds, but have sub-second precision. And those should compare as equal as long as they don't change at all, AFAIU...

Comment (fabxc) [inline content/docs/querying/functions.md]: If protobuf preserves the representation, then yes - don't know the internals.
We can certainly add a notice later if there are people asking questions about it.

**Pull request merged**